pandas/core/series.py

```diff
@@ -2539,6 +2539,13 @@ def append(self, to_append, ignore_index=False, verify_integrity=False) -> "Seri
             to_concat.extend(to_append)
         else:
             to_concat = [self, to_append]
+        for x in to_concat[1:]:
+            if isinstance(x, (pd.DataFrame,)):
+                msg = (
+                    f"to_append should be a Series or list/tuple of Series, "
+                    f"got {type(to_append).__name__}"
```

Comment:
may make the error message clearer in the case where the sequence contains a DataFame
```suggestion
                    f"got {type(x).__name__}"
```